### DISCUSSION TITLE
Add creation timestamp to the Alias card

### DISCUSSION THREAD

**Discussion by WhamboMPS** [simple-login/app#576] [Feature Requests]
Add a field to each [Alias](https://app.simplelogin.io/dashboard/) card that shows a [24-hour clock](https://en.wikipedia.org/wiki/24-hour_clock) based timestamp for its initial creation in the SimpleLogin database. Recommend that [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) be used for the timestamp format, converted to the time zone of the [user agent](https://en.wikipedia.org/wiki/User_agent) from which the query is being made. For instance:

`Creation Date: YYYY-MM-DD HH:MM TimeZone`

For example,

`Creation Date: 2021-08-13 13:21 PDT`

**Comment by nguyenkims** [comment]
@WhamboMPS the ISO 8601 format isn't really human friendly as it requires the brain to compute :). Do you know in what case displaying this format is useful?

The exact alias creation date is actually exposed in the API and can be included in the alias export.

**Comment by WhamboMPS** [comment]
@nguyenkims, I only suggested using the ISO standard 'cause, well, it's _the_ standard. ;-)

Let's back up for a moment: I only suggested the ISO format to eliminate date representation confusion in various regions (e.g. `9/6/2021` in the U.S. is September 6th but in in France I believe that would be June 9th.) I suspect that you don't need to choose a universal format and can simply let the user's app or browser's regional settings display the date in an appropriate manner. Getting something that is correctly interpreted by the user is all that is important.

**Reply by nguyenkims** [reply]
I think the current approach of displaying a relative time (e.g. 3 minutes ago) is more user-friendly. I guess most users don't needthe exact moment an alias is created, except for debugging purposes in which case we can add the ISO datetime in the exported alias. What do you think?

**Reply by WhamboMPS** [reply]
Oh jeez, what a bonehead I have been, @nguyenkims. What I am requesting is actually on the card already! I was focused on the most recent usage timestamp (in cyan in the screenshot below) while the creation timestamp was there all the time (in magenta).

On relative vs. specific timestamp format my preference is for exact values (down to the day for something like this). However, I think this is a personal preference issue and should be left to SL's designer's preference. For what it's worth my workaround is shown in yellow.

![SL Creation Date Screen Image](https://user-images.githubusercontent.com/88680678/134215636-e1b8bf1c-e818-480a-aa1d-7bad82e0ff53.png)


I believe you can close this issue as not to be implemented (or at a minimum prioritized very low if you think exact dates are better). If you do convert to exact dates, I would recommend using the OS's date format settings for the display mask.

**Reply by nguyenkims** [reply]
Yeah date display is a bit tricky: for a recent date, relative time is more intuitive (e.g. 5 minutes ago) whereas, for something like "2 months 10 days ago", exact date is easier to understand. I'll add the "done" tag for this discussion and we can open it up again in the future.
